examples/server/server.cpp

```diff
@@ -3353,10 +3353,14 @@ static void log_server_request(const httplib::Request & req, const httplib::Resp
         return;
     }
 
-    LOG_INF("request: %s %s %s %d\n", req.method.c_str(), req.path.c_str(), req.remote_addr.c_str(), res.status);
+    try {
+        LOG_INF("request: %s %s %s %d\n", req.method.c_str(), req.path.c_str(), req.remote_addr.c_str(), res.status);
 
-    LOG_DBG("request:  %s\n", req.body.c_str());
-    LOG_DBG("response: %s\n", res.body.c_str());
+        LOG_DBG("request:  %s\n", req.body.c_str());
+        LOG_DBG("response: %s\n", res.body.c_str());
+    } catch (const std::exception & e) {
+        LOG_ERR("failed to log request/response: %s\n", e.what());
+    }
```

Comment:
Do any of these throw?

Reply:
No, but this is mostly for future-proof.

Indeed, I was a bit doubt when adding this try..catch because I want to communicate to whoever in the future touching this code that "exception is not handled here". Another solution is to just add a `TODO`, but not sure if it's enough because people may not even read it.

Reply:
OK I replaced the try catch with a comment